config/README.md

```diff
@@ -0,0 +1,125 @@
+| filter-tags       | Array of tags to filter snapshots.                    | No tags filter  ||
+
+### Backup Options
+
+| Attribute         | Description                                            | Default Value  | Example Value |
+|-------------------|--------------------------------------------------------|----------------|---------------|
+| as-path                | Specifies the path for the backup when the source contains a single path. | Not set (Source-specific option) ||
+| description            | Description for the backup.                           | Not set        ||
+| description-from       | Path to a file containing the description for the backup. | Not set    ||
+| delete-never           | If true, never delete the backup.                     | false          ||
+| delete-after           | Time duration after which the backup will be deleted. | Not set        ||
+| exclude-if-present     | Array of filenames to exclude from the backup if they are present. | Not set (Source-specific option) ||
+| force                  | If true, forces the backup even if no changes are detected. | Not set (Source-specific option) ||
+| glob-file              | Array of glob files specifying additional files to include in the backup. | Not set (Source-specific option) ||
+| group-by               | Grouping strategy for the backup.                     | Not set (Source-specific option) ||
+| host                   | Host name for the backup.                             | Not set (Source-specific option) ||
+| ignore-ctime           | If true, ignores file change time (ctime) for the backup. | Not set (Source-specific option) ||
+| ignore-inode           | If true, ignores file inode for the backup.           | Not set (Source-specific option) ||
+| label                  | Label for the backup.                                 | Not set        ||
+| one-file-system        | If true, only backs up files from the same filesystem as the source. | Not set (Source-specific option) ||
+| parent                 | Parent snapshot ID for the backup.                    | Not set (Source-specific option) ||
+| stdin-filename         | File name to be used when reading from stdin.         | Not set (Source-specific option) ||
+| tag                    | Array of tags for the backup.                         | Not set        ||
+| with-atime             | If true, includes file access time (atime) in the backup. | Not set (Source-specific option) ||
+
+### Backup Sources
+
+| Attribute         | Description                                            | Default Value  | Example Value |
+|-------------------|--------------------------------------------------------|----------------|---------------|
+| git-ignore             | If true, use .gitignore rules to exclude files from the backup in the source directory. | true ||
+| source                 | Source directory or file to back up.                  | Not set        ||
```

Comment:
Maybe we should add here, that the option given here for the source are used, when the backup command is used to backup this specific source.